packages/expo/src/Speech/ExponentSpeech.web.ts

```diff
@@ -0,0 +1,66 @@
+import { SpeechOptions } from './Speech.types';
+
+export default {
+  get name(): string {
+    return 'ExponentSpeech';
+  },
+  async speak(id: string, text: string, options: SpeechOptions): Promise<SpeechSynthesisUtterance> {
+    const { SpeechSynthesisUtterance } = global;
```

Comment:
```suggestion
    const { SpeechSynthesisUtterance } = window;
```

It's defined as on Window -> https://w3c.github.io/speech-api/#tts-section